src/graphql/trusted_domain.rs

```diff
@@ -63,6 +63,29 @@ impl TrustedDomainMutation {
         Ok(name)
     }
 
+    /// Update a trusted domain, returning the new value.
```

Comment:
Rather than the imperative form, the third-person singular form of the verb is preferred.

Reply:
I finished editing it.
Thank you.